### ISSUE TITLE
Training loss not decreasing

### ISSUE THREAD

**Issue by renanrojasg** [CYang0515/pytorch_ssn#6]
Hi,

I ran your SSN Pytorch implementation **with the default parameters**. It looks like the loss function is not decreasing when training on the BSDS. Attached is a plot of both losses after 475 epochs. No changes were made to the code, except for some Tensorboard details.

Were you able to get a better training? Maybe there's something I'm doing wrong. The only warning I get during runtime is related to the scipy zoom function.

![20200606_SSN_loss](https://user-images.githubusercontent.com/45795596/83944676-f8587300-a7ca-11ea-8c78-0d28feacf187.png)

Thanks a lot.

**Comment by CYang0515** [comment]
The pos_loss will increase gradually, and the total loss will decrease. You can test the results.
